app.js

```diff
@@ -0,0 +1,7 @@
+import * as config from './config/taskInfo.json';
+import { Product, User } from 'models';
+
+console.log(config.name);
```

Comment:
it's better to use an logging abstraction tool. Like this https://www.npmjs.com/package/winston

Reply:
why do you need config here, are you going to leverage it?

Reply:
It's a part of the task